File: solutions/erlang/leap/1/src/leap.erl
-module(leap).

-export([leap_year/1]).


leap_year(_Year) -> 
    if
        _Year rem 4 /= 0 ->
            false;
        _Year rem 400 == 0 ->
            true;
        _Year rem 100 == 0 ->
            false;
        true ->
            true
    end.

File: solutions/go/leap/1/leap.go
// Package leap provides utility functions that tell wether a year is leap year or not
package leap

// IsLeapYear returns true if the input is a leap year, false otherwise
func IsLeapYear(year int) bool {
	return year % 4 == 0 && year % 100 != 0 || year % 400 == 0
}
